DESCRIPTION

```diff
@@ -70,8 +70,12 @@ Suggests:
     IRdisplay,
     processx,
     plotlyGeoAssets,
-    forcats
+    forcats,
+    httpuv
 LazyData: true
-RoxygenNote: 6.0.1.9000
+RoxygenNote: 6.1.0
 Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
+Remotes:
+    cpsievert/diffobj@css,
+    cpsievert/vdiffr@plotly
```

Comment:
We should probably remove this bit and make a note somewhere that you need these branches for visual testing (at the moment)